photo-app/app/models/payment.rb

```diff
@@ -0,0 +1,19 @@
+class Payment < ApplicationRecord
+  attr_accessor :card_number, :card_cvv, :card_expires_month, :card_expires_year
+  belongs_to :user
+  def self.month_options
+    Date::MONTHNAMES.compact.each_with_index.map { |name, i| ["#{i+1} - #{name}", i+1]}
+  end
+
+  def self.year_options
+    (Date.today.year..(Date.today.year+10)).to_a
+  end
+
+  def process_payment
+    customer = Stripe::Customer.create email: email, card: token
+    Stripe::Charge.create customer: customer.id,
+    amount: 1000,
+    description: 'Premium',
+    currency: 'usd'
```

Comment:
Look at this properly Does it clearly illustrate where is the start & end of the argument you passing in? Where is the method? Think please. THink of others. Not just yourself. Don't be so selfish